CHANGELOG.md

```diff
@@ -2,6 +2,55 @@
 
 Notable changes to QMCPACK will be documented in this file. 
 
+## [Unreleased]
+
+### Notes
+
+This section describes improvements made to the the development branch
+since the last release.
+
+### QMCPACK updates
+
+* Improved population control during DMC equilibration. Reduces variance on larger runs.
+* All particle move VMC and DMC algorithms enabled, tests added
+* Reptation Monte Carlo (RMC) enabled, tests added
+* Significantly improved AFQMC implementation
+* Added NiO based VMC and DMC performance tests and description in manual. Files accessed via QMC_DATA
+* Added DMC tests with locality and t-moves aapproximations
+* Added AFQMC tests
+* Added test of real space QMC restart capabilities
+* Added tests for several estimators
+* Added unit test for DMC walker propagation, effective core potentials, OhmmsPETE
+* To avoid filesystem limitations, QMC_SYMLINK_TEST_FILES can be set to symlink (1) or copy test files (0)
+* Status of t-moves now printed
+* Fixed mixed precision Ceperley force evaluation
+* Many updated tests to improve statistical reliability. Removed flux estimator from short tests because they were not reliable enough
+* Tests that rely on non-standard python modules that are not available are skipped
+* Error trap jastrow factors with cutoff radii larger than Wigner Seitz radius
+* Bugfix: 2-body jastrow on GPU with no alternate spin particles
```

Comment:
This more error trapping. Prevent users adding correlation terms on non-existing electron pairs on both CPU and GPU. The bug fix on GPU is only a side effect.